cpp/arcticdb/CMakeLists.txt

```diff
@@ -524,12 +528,13 @@ endif()
 # on their origin (i.e. vendored source or packages on conda-forge).
 if(${ARCTICDB_USING_CONDA})
 
-    # We must use dynamic libraries.
+    # We must use dynamic libraries, except for xxHash, which must be statically linked.
```

Comment:
```suggestion
    # We must use dynamic libraries, except for xxHash, which must be statically linked
    # since `XXH64_state_t` which is used ArcticDB is only available for static linkage.
```

Reply:
Rephrased